docs/spark-writes.md

```diff
@@ -203,6 +203,28 @@ WHERE EXISTS (SELECT oid FROM prod.db.returned_orders WHERE t1.oid = oid)
 
 For more complex row-level updates based on incoming data, see the section on `MERGE INTO`.
 
+### `Writing to Branches`
```

Comment:
why do we need backquotes around the title?

Reply:
We don't good catch, I'm removing